src/onnx-mlir.cpp

```diff
@@ -61,9 +61,10 @@ int main(int argc, char *argv[]) {
 
   mlir::OwningOpRef<mlir::ModuleOp> module;
   std::string errorMessage;
-  processInputFile(inputFilename, context, module, &errorMessage);
-  if (!errorMessage.empty()) {
-    printf("%s\n", errorMessage.c_str());
+  int rc = processInputFile(inputFilename, context, module, &errorMessage);
+  if (rc != 0) {
+    if (!errorMessage.empty())
+      printf("%s\n", errorMessage.c_str());
```

Comment:
Suggest to print the error message to stderr rather than stdout - using fprintf(stderr, ....);

Reply:
For C++ code, the more C++-ish way is probably to use std:cerr/std:cout, etc.

Reply:
very good, used prior code, will switch to `std::stderr`